apps/verify/translations/src/en/fields.json

```diff
@@ -14,15 +14,15 @@
   },
   "brp": {
     "label": "Biometric residence permit (BRP) number",
-    "hint": "This must match what is on your BRP card. For example, RZ1234567"
+    "hint": "This must match what is on your BRP card. For example, ZU1234567"
```

Comment:
ZU1234567 is not a valid pattern despite it being on the graphic.

It has to be [R][A-Z][digit or X][6 digits]